scripts/sync_mephisto_task.py

```diff
@@ -11,12 +11,14 @@
 """
 
 import os
+import sys
 import re
 import json
 from mephisto.operations.utils import get_root_dir
+from mephisto.operations.logger_core import format_loud
 
 ROOT_DIR = get_root_dir()
-PATTERN = r'(CURR_MEPHISTO_TASK_VERSION = "[0-9a-zA-Z.]*")'
+PATTERN = r'(CURR_MEPHISTO_TASK_VERSION = "([0-9a-zA-Z.]*)")'
```

Comment:
Parenthesis will allow us to extract the version number out easily programatically